charts/bindplane/values.yaml

```diff
@@ -301,3 +308,13 @@ dev:
       name: ghcr.io/observiq/observiq-otel-collector
       tag: latest
     labels: "configuration=test"
+  
+  # Manages a prometheus deployment for development purposes. Production
+  # deployments should use a proper prometheus backend deployed outside
+  # of this chart.
+  prometheus:
+    # Whether or not prometheus should be deployed.
+    create: false
+    image:
+      name: prom/prometheus
+      tag: v2.48.0
```

Comment:
We are currently locked to v2.47.2 in Bindplane https://github.com/observIQ/bindplane-op-enterprise/blob/release/v1.38.0/PROMETHEUS_VERSION